inc/command/class-command.php

```diff
@@ -188,14 +202,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
+		$use_chassis = $input->getOption( 'chassis' );
+		$tests_folder = $module !== 'project' ? "altis/$module/tests" : 'tests';
+		$project_subdomain = $this->get_project_subdomain();
+
+		// Write the default config.
+		$config = [
+			'paths' => [
+				'tests' => $tests_folder,
+				'output' => 'altis/dev-tools/tests/_output',
+				'data' => 'altis/dev-tools/tests/_data',
+				'support' => 'altis/dev-tools/tests/_support',
+			],
+			'actor_suffix' => 'Tester',
+			'extensions' => [
+				'enabled' => [
+					'Codeception\Extension\RunFailed',
+				],
+				'commands' => [
+					'Codeception\Command\GenerateWPUnit',
+					'Codeception\Command\GenerateWPRestApi',
+					'Codeception\Command\GenerateWPRestController',
+					'Codeception\Command\GenerateWPRestPostTypeController',
+					'Codeception\Command\GenerateWPAjax',
+					'Codeception\Command\GenerateWPCanonical',
+					'Codeception\Command\GenerateWPXMLRPC',
+				],
+			],
+			'modules' => [
+				'config' => [
+					'WPDb' => [
+						'dsn' => '%TEST_SITE_DB_DSN%',
+						'user' => '%TEST_SITE_DB_USER%',
+						'password' => '%TEST_SITE_DB_PASSWORD%',
+						'dump' => '%TEST_SITE_DB_DUMP%',
+						'populate' => true,
+						'cleanup' => true,
+						'waitlock' => 10,
+						'url' => '%TEST_SITE_WP_URL%',
+						'urlReplacement' => false,
+						'tablePrefix' => '%TEST_SITE_TABLE_PREFIX%',
+					],
+					'WPBrowser' => [
+						'url' => '%TEST_SITE_WP_URL%',
+						'adminUsername' => '%TEST_SITE_ADMIN_USERNAME%',
+						'adminPassword' => '%TEST_SITE_ADMIN_PASSWORD%',
+						'adminPath' => '%TEST_SITE_WP_ADMIN_PATH%',
+						'headers' => [
+							'X_TEST_REQUEST' => 1,
+							'X_WPBROWSER_REQUEST' => 1,
+						],
+					],
+					'WPWebDriver' => [
+						'url' => '%TEST_SITE_WP_URL%',
+						'adminUsername' => '%TEST_SITE_ADMIN_USERNAME%',
+						'adminPassword' => '%TEST_SITE_ADMIN_PASSWORD%',
+						'adminPath' => '%TEST_SITE_WP_ADMIN_PATH%',
+						'browser' => $run_headless_browser,
+						'host' => '172.17.0.1',
+						'port' => '4444',
+						'window_size' => false, // disabled for Chrome driver.
+						'capabilities' => [
+							'chromeOptions' => [
+								'args' => [
+									'--headless',
+									'--disable-gpu',
+									'--proxy-server=\'direct://\'',
+									'--proxy-bypass-list=*',
+									'--user-agent=wp-browser',
```

Comment:
That's a nice idea, I wondered about that myself.

Reply:
Ah, now I remember why I didn't go through with it, as we can only _specify_ the user string, rather than _amend_ it, eg: I can't get the actual version used by the Chrome browser. 

I can simulate the user agent string, but the version wouldn't be correct, which doesn't seem to be a huge problem.

Reply:
We force the user agent to a specific string in cloud environments, as it's not passed through from CloudFront: https://github.com/humanmade/altis-cloud/blob/e0e5162c8942637fac9db02c56d4d09ac2fa5c2c/inc/namespace.php#L172

Reply:
Ah that's true - for analytics purposes we read it client side. Lets not overengineer this then - what you have is perfect.

Reply:
Well, this is the client-side of things, these are the startup parameters for the browser that'll carry out the acceptance tests. And it's not related to the cloud environment, presumably the referenced line in https://github.com/humanmade/altis-cloud/blob/e0e5162c8942637fac9db02c56d4d09ac2fa5c2c/inc/namespace.php#L172 wouldn't be reached as the browser will have sent an actual user agent string, not an empty one nor `Amazon CloudFront`.

I went ahead and updated the user agent strings, and their handling, to _include_ `wp-browser` rather than _be_ `wp-browser` which works just as good.